### PR TITLE
Updating deploy scripts and rvol bytecode

### PR DESCRIPTION
**Pull request by fqlx**
- Updating deploy scripts to use `chainId`
- Updating rvol bytecode to commit sha `11a34ecb5e43c98a31c20ac2da33e15a8d677449`
- Fixing eslint in package.json
- Move constants up a folder so migration doesn't fail.  scripts/deploy/utils/constants.ts -> scripts/utils/constants.ts

```
$ npx hardhat deploy --network fuji
00 - Deploying ManualVolOracle on fuji
ManualVolOracle @ 0x8c58dDE5156DE3b3Fb93664DccBc1BeB1d5Ac923
01 - Deploying Theta Vault logic on fuji
RibbonThetaVaultLogic @ 0x5d5Acc6D05f1c675458fCf9847EA64CEBdFDb5f1
02 - Deploying ETH Call Theta Vault on fuji
RibbonThetaVaultETHCall @ 0x9e63B8E9291507EA9cEBabB5b7c195bD478E7CDd
03 - Deploying WBTC Call Theta Vault on fuji
RibbonThetaVaultWBTCCall @ 0xAB0D710a57C259f3bE38Ca33493729d5FCD3335E
04 - Deploying Delta Vault logic on fuji
RibbonDeltaVaultLogic @ 0xadb6BbA485390b5581d9bB178158fda23a417e57
05 - Deploying ETH Call Delta Vault on fuji
RibbonDeltaVaultETHCall @ 0x14f4f0B7a1dBF7dffD911D3DafE253eD468f1413
06 - Skipping deployment AAVE Call Theta Vault on fuji
07 - Skipping deployment of Theta Vault stETH logic on fuji because no stEth on Avax
08 - Skipping deployment stETH Call Theta Vault on fuji because no stEth on Avax
```

### CODE REVIEW

File: scripts/deploy/08_steth_call_theta_vault.ts [hunk 1]
```diff
@@ -44,22 +46,26 @@ const main = async ({
...
-    MANAGEMENT_FEE,
-    PERFORMANCE_FEE,
-    "Ribbon stETH Theta Vault",
-    "rstETH-THETA",
-    pricer.address,
-    strikeSelection.address,
-    PREMIUM_DISCOUNT,
-    AUCTION_DURATION,
+    {
+      _owner: owner,
```
Comment (chuddster) [inline scripts/deploy/08_steth_call_theta_vault.ts:50]: for some of these you prepend with param name, and for some vault deploy scripts like delta you dont

File: scripts/utils/constants.ts [hunk 2]
```diff
@@ -1,21 +1,11 @@
 /**
  * Addresses
  */
-export const ETH_USDC_POOL = "0x8ad599c3A0ff1De082011EFDDc58f1908eb6e6D8";
-export const MAINNET_ETH_ORACLE = "0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419";
-export const KOVAN_ETH_ORACLE = "0x9326BFA02ADD2366b30bacB125260Af641031331";
-export const KOVAN_WETH = "0xd0A1E359811322d97991E03f863a0C30C2cF029C";
```
Comment (chuddster) [inline scripts/utils/constants.ts:7]: why do we remove these

Comment (fqlx) [reply scripts/utils/constants.ts:7]: These variables are already defined in `constants/constants.ts` and we should only have 1 source of truth, not define the variables in multiple files.

Ideally, I would like to remove `scripts/utils/constants.ts` in the future and have it all defined in constants/constants.ts`

File: scripts/deploy/08_steth_call_theta_vault.ts (hunk 1 above)
Comment (fqlx) [reply scripts/deploy/08_steth_call_theta_vault.ts]: Good catch, thanks.

**Review by chuddster — APPROVED** [review]

**Pull request merged**